Source/WebInspectorUI/UserInterface/Views/DOMTreeContentView.js

```diff
@@ -694,6 +702,97 @@ WI.DOMTreeContentView = class DOMTreeContentView extends WI.ContentView
         this._showPrintStylesChanged();
     }
 
+    _handleOverrideUserPreferencesButtonClicked()
+    {
+        if (this._userPreferencesOverridesPopover) {
+            this._userPreferencesOverridesPopover.dismiss();
+            this._userPreferencesOverridesPopover = null;
+            return;
+        }
+
+        let contentElement = document.createElement("div");
+        contentElement.classList.add("user-preferences-content");
+
+        let createSelectElement = (label, preference, values = [], defaultValue) => {
```

Comment:
Style: no space between `[` `]`

also, it looks like every caller of this provides values for that (and we should expect that) so I don't think we even need the optional fallback or the `!values.length` check